cmake/pml_packaging.cmake

```diff
@@ -0,0 +1,49 @@
+
+
+# Set general Cpack Variables for package description
+set(CPACK_GENERATOR "TGZ")
+set(CPACK_ARCHIVE_COMPONENT_INSTALL 1)
+set(CPACK_PACKAGE_NAME "PlaidML")
+set(CPACK_PACKAGE_VENDOR "Intel Corp")
+set(CPACK_PACKAGE_CONTACT "Intel")
+set(CPACK_PACKAGE_VERSION ${PLAIDML_VERSION})
+set(CPACK_COMPONENTS_IGNORE_GROUPS 1)
+set(CPACK_COMPONENTS_ALL devkit)
+
+set_target_properties(mlir_runner_utils PROPERTIES INTERFACE_LINK_LIBRARIES "")
+
+install(TARGETS plaidml_plaidml
+                omp
+                mlir_runner_utils
+        EXPORT ${PROJECT_NAME}_Targets
+        DESTINATION lib
+        COMPONENT devkit)
```

Comment:
We might try installing files rather than targets. I haven't had much luck with getting targets to work.